packages/wrangler/src/deploy/deploy.ts

```diff
@@ -676,17 +676,10 @@ See https://developers.cloudflare.com/workers/platform/compatibility-dates for m
 			limits: config.limits,
 		};
 
-		// As this is not deterministic for testing, we detect if in a jest environment and run asynchronously
-		// We do not care about the timing outside of testing
-		const bundleSizePromise = printBundleSize(
+		await printBundleSize(
```

Comment:
Contrary to the comment that was previously here—while we don't care about the value, we do care about the order of log statements for snapshot testing (and `printBundleSize()` writes to stdout)